core/src/main/java/com/hevelian/identity/core/exc/ImageReadException.java

```diff
@@ -0,0 +1,23 @@
+package com.hevelian.identity.core.exc;
+
+public class ImageReadException extends RuntimeException {
```

Comment:
This shoul be an Exception, not a Runtime exception. That was a different between this class an ImageRetrievalException...

Reply:
Fixed